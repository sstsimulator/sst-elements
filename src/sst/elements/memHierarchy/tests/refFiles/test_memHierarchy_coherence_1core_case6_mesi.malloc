Number-of-pages: 5
alloc_unit_: 1024
init_: 1
shift_: 10
addr: 0
0 0xc00000000000000 0x1400000000000000 0 0 0 0x30000000 0x3c00000038000000 0 0x4c00000000000000 0x5400000000000000 0x58000000 0x6400000060000000 0x68000000 0x7400000070000000 0x78000000 0x8400000080000000 0x88000000 0x9400000090000000 0x9c00000000000000 0xa400000000000000 0xac000000a8000000 0xb0000000 0xbc000000b8000000 0xc4000000c0000000 0xcc00000000000000 0xd0000000 0 0xe0000000 0xec00000000000000 0 0xf8000000 0x401000000010000 0x8010000 0 0x1c01000000000000 0x2401000000000000 0x2c01000000000000 0x3401000030010000 0x38010000 0x4401000040010000 0x4c01000000000000 0x50010000 0x5c01000000000000 0x6401000000000000 0x6c01000000000000 0x7401000070010000 0x7c01000000000000 0x8401000080010000 0x8c01000000000000 0x9401000000000000 0x98010000 0xa4010000a0010000 0xa8010000 0xb0010000 0xb8010000 0xc0010000 0xcc01000000000000 0xd0010000 0 0xe4010000e0010000 0xec010000e8010000 0xf4010000f0010000 0xfc01000000000000 0 0xc02000000000000 0 0x1c02000018020000 0x2402000000000000 0 0x3402000000000000 0x3c02000038020000 0x4402000000000000 0x4c02000048020000 0 0 0x6402000000000000 0x6c02000068020000 0x70020000 0x7c02000000000000 0x8402000000000000 0 0x9402000090020000 0 0 0xa8020000 0xb0020000 0xbc02000000000000 0 0 0xd402000000000000 0xdc02000000000000 0xe4020000e0020000 0xec020000e8020000 0xf4020000f0020000 0xf8020000 0 0x8030000 0x1403000000000000 0x1c03000018030000 0x2403000020030000 0x2c03000000000000 0 0x38030000 0x4403000000000000 0x48030000 0x50030000 0x58030000 0x60030000 0x68030000 0x70030000 0x7c03000078030000 0x8403000000000000 0x88030000 0 0 0xa0030000 0xac030000a8030000 0xb4030000b0030000 0xbc03000000000000 0xc0030000 0xcc03000000000000 0xd0030000 0xd8030000 0xe403000000000000 0 0 0xfc030000f8030000
addr: 0x400
0x404000000000000 0 0x1404000010040000 0x18040000 0x2404000000000000 0 0x3404000030040000 0x3c04000038040000 0x4404000040040000 0x48040000 0x50040000 0x5c04000000000000 0x6404000060040000 0x6c04000068040000 0x7404000000000000 0x7c04000078040000 0 0x8c04000000000000 0x90040000 0x9c04000098040000 0 0xa8040000 0xb4040000b0040000 0xb8040000 0xc4040000c0040000 0xc8040000 0xd404000000000000 0xdc040000d8040000 0 0xec04000000000000 0xf404000000000000 0xf8040000 0x50000 0xc05000000000000 0x1405000000000000 0x1c05000000000000 0x2405000020050000 0 0x3405000000000000 0 0x40050000 0x4c05000048050000 0x5405000050050000 0x5c05000000000000 0x6405000000000000 0x6c05000068050000 0x7405000070050000 0x78050000 0x8405000000000000 0x8c05000000000000 0x9405000090050000 0x98050000 0xa0050000 0xa8050000 0xb0050000 0xbc05000000000000 0 0xc8050000 0xd405000000000000 0xd8050000 0 0xec05000000000000 0xf0050000 0 0x406000000060000 0xc06000000000000 0 0 0x2406000020060000 0x2c06000028060000 0 0x38060000 0x4406000000000000 0x48060000 0x5406000000000000 0x5c06000058060000 0x60060000 0x68060000 0x70060000 0x7c06000078060000 0 0x8c06000088060000 0x90060000 0x9c06000098060000 0xa0060000 0 0 0xb8060000 0 0xcc060000c8060000 0xd0060000 0 0 0 0xf406000000000000 0xfc060000f8060000 0 0x8070000 0x10070000 0x1c07000018070000 0x20070000 0x2c07000028070000 0 0x38070000 0x4407000040070000 0x4c07000048070000 0 0x58070000 0x60070000 0x6c07000068070000 0x7407000070070000 0x7c07000000000000 0 0 0x9407000090070000 0x9c07000098070000 0 0xa8070000 0xb4070000b0070000 0xbc070000b8070000 0xc0070000 0xc8070000 0 0xdc07000000000000 0xe0070000 0 0xf0070000 0
addr: 0x800
0x80000 0x8080000 0x1408000010080000 0x18080000 0x2408000000000000 0x28080000 0x3408000000000000 0x3c08000000000000 0x40080000 0x4c08000048080000 0x5408000050080000 0x58080000 0x60080000 0x68080000 0x7408000070080000 0x7c08000078080000 0x8408000080080000 0x88080000 0 0x9c08000098080000 0xa0080000 0xac080000a8080000 0xb4080000b0080000 0xbc08000000000000 0 0xcc08000000000000 0xd0080000 0 0xe4080000e0080000 0xe8080000 0xf0080000 0xf8080000 0 0 0x1409000000000000 0 0x2409000000000000 0x28090000 0x3409000030090000 0x3c09000000000000 0x40090000 0x4c09000048090000 0 0x58090000 0 0 0x7409000070090000 0x7c09000000000000 0x8409000080090000 0 0x90090000 0x9c09000098090000 0xa4090000a0090000 0 0xb4090000b0090000 0xbc090000b8090000 0xc409000000000000 0xcc09000000000000 0xd4090000d0090000 0xd8090000 0xe0090000 0 0xf4090000f0090000 0xfc090000f8090000 0x40a0000000a0000 0x80a0000 0x100a0000 0x1c0a000000000000 0 0x2c0a0000280a0000 0x300a0000 0x3c0a000000000000 0x440a000000000000 0x4c0a000000000000 0x500a0000 0x5c0a000000000000 0x640a0000600a0000 0 0 0x780a0000 0x800a0000 0x8c0a000000000000 0 0x980a0000 0xa00a0000 0xac0a000000000000 0xb40a000000000000 0xb80a0000 0xc00a0000 0xc80a0000 0xd40a000000000000 0xdc0a000000000000 0xe00a0000 0xe80a0000 0xf00a0000 0 0 0xc0b0000080b0000 0 0x1c0b0000180b0000 0 0x2c0b000000000000 0x340b0000300b0000 0x3c0b000000000000 0x400b0000 0x480b0000 0 0x5c0b0000580b0000 0x640b000000000000 0x6c0b000000000000 0x740b000000000000 0x780b0000 0x840b0000800b0000 0 0x940b000000000000 0x980b0000 0xa00b0000 0xa80b0000 0xb40b000000000000 0xbc0b000000000000 0 0xc80b0000 0xd40b0000d00b0000 0xdc0b0000d80b0000 0xe40b0000e00b0000 0xec0b0000e80b0000 0 0xfc0b0000f80b0000
addr: 0xc00
0x40c000000000000 0xc0c0000080c0000 0x140c0000100c0000 0x1c0c0000180c0000 0x200c0000 0x280c0000 0x300c0000 0x3c0c000000000000 0x440c0000400c0000 0x4c0c000000000000 0 0x580c0000 0 0 0 0x7c0c0000780c0000 0x800c0000 0x8c0c000000000000 0x900c0000 0x980c0000 0xa40c000000000000 0xac0c0000a80c0000 0xb40c0000b00c0000 0 0xc40c0000c00c0000 0xc80c0000 0xd40c0000d00c0000 0 0xe40c000000000000 0xec0c000000000000 0xf00c0000 0xfc0c0000f80c0000 0 0xc0d000000000000 0x140d000000000000 0 0x240d0000200d0000 0 0x340d0000300d0000 0 0x440d000000000000 0x4c0d000000000000 0x540d0000500d0000 0x580d0000 0x640d0000600d0000 0x680d0000 0x700d0000 0x7c0d000000000000 0x840d0000800d0000 0x8c0d000000000000 0 0x9c0d000000000000 0 0xac0d0000a80d0000 0 0xb80d0000 0 0 0xd00d0000 0xdc0d000000000000 0 0xec0d000000000000 0xf40d0000f00d0000 0xf80d0000 0x40e000000000000 0xc0e0000080e0000 0x140e000000000000 0 0 0 0 0 0x440e000000000000 0x480e0000 0x500e0000 0 0x600e0000 0x6c0e000000000000 0x740e000000000000 0x7c0e000000000000 0x800e0000 0x8c0e0000880e0000 0 0 0xa00e0000 0xac0e000000000000 0xb40e0000b00e0000 0xbc0e000000000000 0xc40e0000c00e0000 0xc80e0000 0xd00e0000 0xdc0e0000d80e0000 0xe40e0000e00e0000 0 0xf40e000000000000 0xfc0e000000000000 0x40f000000000000 0x80f0000 0x140f0000100f0000 0 0 0x2c0f000000000000 0x300f0000 0x380f0000 0x400f0000 0x480f0000 0 0x5c0f0000580f0000 0 0x6c0f000000000000 0 0x7c0f0000780f0000 0x800f0000 0x8c0f000000000000 0x940f000000000000 0x980f0000 0xa40f0000a00f0000 0 0xb00f0000 0xb80f0000 0xc40f0000c00f0000 0 0xd40f000000000000 0xd80f0000 0xe00f0000 0xec0f000000000000 0 0xfc0f000000000000
addr: 0x1000
0 0xc10000008100000 0x1410000010100000 0 0 0x2c10000000000000 0x30100000 0 0x40100000 0x4c10000048100000 0x5410000050100000 0x58100000 0x60100000 0x6c10000000000000 0x70100000 0x7c10000000000000 0x8410000000000000 0x8c10000000000000 0x9410000000000000 0 0xa0100000 0xac100000a8100000 0xb410000000000000 0xb8100000 0xc0100000 0 0xd410000000000000 0xd8100000 0xe4100000e0100000 0xec10000000000000 0xf410000000000000 0xfc100000f8100000 0x411000000110000 0x8110000 0x10110000 0x1c11000018110000 0x2411000020110000 0x2c11000028110000 0x3411000000000000 0x3c11000000000000 0x4411000000000000 0x4c11000048110000 0 0x5c11000058110000 0 0x68110000 0x70110000 0x7c11000000000000 0x8411000080110000 0 0x9411000000000000 0x9c11000000000000 0xa4110000a0110000 0 0 0xb8110000 0xc0110000 0xcc11000000000000 0xd411000000000000 0xdc110000d8110000 0xe0110000 0xe8110000 0xf4110000f0110000 0xf8110000 0 0xc12000000000000 0x1412000010120000 0x1c12000018120000 0x2412000020120000 0 0x3412000000000000 0x3c12000038120000 0 0x4c12000000000000 0 0x5c12000000000000 0x6412000060120000 0x6c12000000000000 0x7412000070120000 0x7c12000000000000 0x8412000080120000 0x8c12000000000000 0x90120000 0x9c12000000000000 0xa0120000 0xa8120000 0xb0120000 0xbc12000000000000 0xc4120000c0120000 0xc8120000 0xd0120000 0 0 0 0 0xfc120000f8120000 0 0x8130000 0x1413000000000000 0x1c13000018130000 0x2413000000000000 0x28130000 0 0x3c13000000000000 0x4413000000000000 0x48130000 0x50130000 0x58130000 0 0x6c13000000000000 0x70130000 0x7c13000078130000 0x80130000 0x8c13000088130000 0x90130000 0 0xa0130000 0xac130000a8130000 0xb413000000000000 0xb8130000 0xc0130000 0xc8130000 0xd0130000 0xdc130000d8130000 0xe4130000e0130000 0xe8130000 0xf4130000f0130000 0xf8130000
