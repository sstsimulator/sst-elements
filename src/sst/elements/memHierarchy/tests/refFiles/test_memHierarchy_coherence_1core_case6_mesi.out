 core.pendCycle : Accumulator : Sum.u64 = 8486646; SumSQ.u64 = 135490778; Count.u64 = 532484; Min.u64 = 0; Max.u64 = 16; 
 core.reads : Accumulator : Sum.u64 = 1405; SumSQ.u64 = 1405; Count.u64 = 1405; Min.u64 = 1; Max.u64 = 1; 
 core.writes : Accumulator : Sum.u64 = 881; SumSQ.u64 = 881; Count.u64 = 881; Min.u64 = 1; Max.u64 = 1; 
 core.flushcaches : Accumulator : Sum.u64 = 76; SumSQ.u64 = 76; Count.u64 = 76; Min.u64 = 1; Max.u64 = 1; 
 core.llsc : Accumulator : Sum.u64 = 69; SumSQ.u64 = 69; Count.u64 = 69; Min.u64 = 1; Max.u64 = 1; 
 core.llsc_success : Accumulator : Sum.u64 = 69; SumSQ.u64 = 69; Count.u64 = 69; Min.u64 = 1; Max.u64 = 1; 
 memory.requests_received_GetS : Accumulator : Sum.u64 = 1405; SumSQ.u64 = 1405; Count.u64 = 1405; Min.u64 = 1; Max.u64 = 1; 
 memory.requests_received_GetSX : Accumulator : Sum.u64 = 69; SumSQ.u64 = 69; Count.u64 = 69; Min.u64 = 1; Max.u64 = 1; 
 memory.requests_received_GetX : Accumulator : Sum.u64 = 0; SumSQ.u64 = 0; Count.u64 = 0; Min.u64 = 0; Max.u64 = 0; 
 memory.requests_received_Write : Accumulator : Sum.u64 = 950; SumSQ.u64 = 950; Count.u64 = 950; Min.u64 = 1; Max.u64 = 1; 
 memory.requests_received_PutM : Accumulator : Sum.u64 = 0; SumSQ.u64 = 0; Count.u64 = 0; Min.u64 = 0; Max.u64 = 0; 
 memory.outstanding_requests : Accumulator : Sum.u64 = 2426868; SumSQ.u64 = 38740732; Count.u64 = 152290; Min.u64 = 1; Max.u64 = 16; 
 memory.latency_GetS : Accumulator : Sum.u64 = 1406659; SumSQ.u64 = 1408320641; Count.u64 = 1405; Min.u64 = 1001; Max.u64 = 1012; 
 memory.latency_GetSX : Accumulator : Sum.u64 = 69083; SumSQ.u64 = 69166113; Count.u64 = 69; Min.u64 = 1001; Max.u64 = 1003; 
 memory.latency_GetX : Accumulator : Sum.u64 = 0; SumSQ.u64 = 0; Count.u64 = 0; Min.u64 = 0; Max.u64 = 0; 
 memory.latency_Write : Accumulator : Sum.u64 = 951126; SumSQ.u64 = 952253650; Count.u64 = 950; Min.u64 = 1001; Max.u64 = 1009; 
 memory.latency_PutM : Accumulator : Sum.u64 = 0; SumSQ.u64 = 0; Count.u64 = 0; Min.u64 = 0; Max.u64 = 0; 
 memory.cycles_with_issue : Accumulator : Sum.u64 = 2424; SumSQ.u64 = 2424; Count.u64 = 2424; Min.u64 = 1; Max.u64 = 1; 
 memory.cycles_attempted_issue_but_rejected : Accumulator : Sum.u64 = 0; SumSQ.u64 = 0; Count.u64 = 0; Min.u64 = 0; Max.u64 = 0; 
 memory.total_cycles : Accumulator : Sum.u64 = 152290; SumSQ.u64 = 23192244100; Count.u64 = 1; Min.u64 = 152290; Max.u64 = 152290; 
Simulation is complete, simulated time: 152.29 us
